app/controllers/verify/sessions_controller.rb

```diff
@@ -20,6 +19,18 @@ def new
     end
 
     def create
+      result = idv_form.submit(profile_params)
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED_FORM, result.to_h)
+
+      if result.success?
+        submit_idv_job
+        redirect_to verify_session_result_path
+      else
+        process_failure
+      end
+    end
+
+    def show
```

Comment:
The name `show` is still throwing me off. When I see `show`, I think of a page that is displaying something to the user, and not doing anything else. Maybe rename this to `result`?

I also have a question about what happens when the vendor times out. Let's say the user submits the initial personal info form, which redirects to `/verify/session/result` while we're waiting for the vendor to respond. If the vendor times out, we are currently failing the user, which is not fair to the user. If the user clicks "Try again" on the modal, they are sent back to the personal info page, but the URL hasn't changed. It's still `/verify/session/result`. So, if the user refreshes the page, it will trigger another attempt, and they will now have 1 less attempt remaining. Can we change this behavior such that the user is not penalized, and is instead presented with a message stating what happened and to try again later?